src/pages/index.js

```diff
@@ -13,15 +15,45 @@ import {
   inputSubtitle,
   profileEditBtn,
   addNewImageBtn,
-  initialCards,
   config,
   userInfoObj,
+  serverToken,
+  avatarForm,
+  avatarOverlay,
+  imageDeleteForm,
 } from "../utils/constants.js";
 const user = new UserInfo(userInfoObj);
-//////////////////////////  validators  //////////////////////////
+const handlerFunctions = {
+  handleDelete: function handleDelete(data) {
+    openPopup(popupImageDelete);
+    popupImageDelete.setDeleteSubmitListener(data);
+  },
+  handleLike: function handleLike(cardData) {
+    apiCall.likeCard(cardData).then((res) => {
+      this.checkLike(res);
```

Comment:
`this` should **not** be used **outside** classes in functions in `index.js` . This **context** is easy to lose. You need to find other ways of passing data.